mailpoet/assets/js/src/automation/editor/components/header/document_actions.tsx

```diff
@@ -48,9 +40,9 @@ function DocumentActions({ children }): JSX.Element {
         {children && (
           <Dropdown
             popoverProps={{
-              anchorRef: titleRef.current,
+              placement: 'bottom-start',
+              anchor: titleRef.current,
             }}
-            position="bottom center"
```

Comment:
More of a note to self to check if the dropdown positioning in the UI somehow changed.